oscm-app-aws-ear/.settings/org.eclipse.wst.common.component

```diff
@@ -0,0 +1,67 @@
+<?xml version="1.0" encoding="UTF-8"?><project-modules id="moduleCoreId" project-version="1.5.0">
+    <wb-module deploy-name="oscm-app-aws">
+        <wb-resource deploy-path="/" source-path="/target/m2e-wtp/ear-resources"/>
+        <wb-resource deploy-path="/" source-path="/src/main/application" tag="defaultRootSource"/>
+        <dependent-module archiveName="oscm-app-aws.jar" deploy-path="/" handle="module:/resource/oscm-app-aws/oscm-app-aws">
+            <dependency-type>uses</dependency-type>
+        </dependent-module>
+        <dependent-module archiveName="oscm-app-common.war" deploy-path="/" handle="module:/resource/oscm-app-common/oscm-app-common">
+            <dependency-type>uses</dependency-type>
+        </dependent-module>
+        <dependent-module archiveName="slf4j-api-1.7.21.jar" deploy-path="/lib" handle="module:/classpath/var/M2_REPO/org/slf4j/slf4j-api/1.7.21/slf4j-api-1.7.21.jar">
+            <dependency-type>uses</dependency-type>
+        </dependent-module>
+        <dependent-module archiveName="oscm-extsvc-master-ef5fda73d2-1.jar" deploy-path="/lib" handle="module:/classpath/var/M2_REPO/com/github/servicecatalog/oscm-interfaces/oscm-extsvc/master-ef5fda73d2-1/oscm-extsvc-master-ef5fda73d2-1.jar">

```

Comment:
Better if all eclipse settings refer to the latest version dependencies as 'master-SNAPSHOT' rather than to particular versions with fix commit hash.